arclightning_frontend/qrcode.js

```diff
@@ -0,0 +1,614 @@
+/**
```

Comment:
I'd prefer to hotlink this rather than copying the entire file due to licensing issues

Reply:
Is [`qrcode.js`](https://raw.githubusercontent.com/davidshimjs/qrcodejs/master/qrcode.js) hosted somewhere that we can hotlink to it other than Github? I don't think we can hotlink directly from Github due to [nosniff](https://blog.github.com/2013-04-24-heads-up-nosniff-header-support-coming-to-chrome-and-firefox/)

Reply:
good catch. This should be fine then